src/pages/components/select/usage.mdx

```diff
@@ -415,6 +426,67 @@ red border, an error icon indicator, and an error message.
 </Column>
 </Row>
 
+## AI presence
+
+Select has a modification that takes on the AI visual styling when the AI slug
+is present in the input. The AI variant functions the same as the normal version
+except with the addition of the AI slug which is both a visual indicator and the
+trigger for the explainability popover.
+
+More information about designing for AI guidelines is coming soon.
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Default">
+
+![Enabled state of default select with AI presence](images/select-usage-AI-presence-default.png)
+
+</Tab>
+
+<Tab label="Fluid">
+
+![Enabled state of fluid select with AI presence](images/select-usage-AI-presence-fluid.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+### Revert to AI
+
+A select can toggle between the AI variant and the non-AI variant depending on
+the user’s interaction. If the user manually overrides the AI-suggested content
+then the input will change from the AI variant to the default variant. Once
```

Comment:
_"content then the input will change from the AI variant to the **non-AI** variant."_